Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -82,12 +84,70 @@ def tap_new
 
             - run: brew test-bot --only-formulae
               if: github.event_name == 'pull_request'
+
+            - name: Upload bottles as artifact
+              if: always() && github.event_name == 'pull_request'
+              uses: actions/upload-artifact@main
+              with:
+                name: bottles
+                path: '*.bottle.*'
+    YAML
+
+    actions_publish = <<~YAML
+      name: brew pr-pull
+      on:
+        pull_request_target:
+          types:
+            - labeled
+      jobs:
+        pr-pull:
+          if: contains(github.event.pull_request.labels.*.name, 'ok')
+          runs-on: ubuntu-latest
+          steps:
+            - name: Set up Homebrew
+              uses: Homebrew/actions/setup-homebrew@master
+
+            - name: Set up git
+              uses: Homebrew/actions/git-user-config@master
+
+            - name: Pull bottles
+              env:
+                HOMEBREW_GITHUB_API_TOKEN: ${{ github.token }}
+                PULL_REQUEST: ${{ github.event.inputs.pull_request }}
+              run: brew pr-pull --debug --tap=$GITHUB_REPOSITORY $PULL_REQUEST
+
+            - name: Push commits
+              uses: Homebrew/actions/git-try-push@master
+              with:
+                token: ${{ github.token }}
+
+            - name: Delete branch
+              if: github.event.pull_request.head.repo.fork == false
+              env:
+                BRANCH: ${{ github.event.pull_request.head.ref }}
+              run: git push --delete origin $BRANCH
     YAML
 
     (tap.path/".github/workflows").mkpath
-    write_path(tap, ".github/workflows/tests.yml", actions)
+    write_path(tap, ".github/workflows/tests.yml", actions_main)
+    write_path(tap, ".github/workflows/publish.yml", actions_publish)
+
+    unless args.no_git?
+      cd tap.path do
+        safe_system "git", "init"
+        safe_system "git", "add", "--all"
+        safe_system "git", "commit", "-m", "Create #{tap} tap"
+      end
+    end
+
     ohai "Created #{tap}"
-    puts tap.path.to_s
+    puts <<~EOS
+      #{tap.path}
+
+      When a pull request making changes to a formula (or formulae) becomes green
+      (all checks passed), then you can publish the built bottles.
+      To do so, label your PR as 'ok' and the workflow will be triggered.
```

Comment:
```suggestion
      To do so, label your PR as #{label} and the workflow will be triggered.
```

and could share the variable above?

Reply:
Even better, it could be an option with default value.

Reply:
Yeh, good idea 👍🏻

Reply:
Although, for a counterpoint: I tend to think it's not worth adding options until someone requests them?

Reply:
I request it :smile: